RangeRequest.bs

```diff
@@ -10,26 +10,218 @@ Editor: Chris Lilley, W3C, https://svgees.us/, w3cid 1438
+
+<h3 id="font-organization-compression">
+Compression</h2>
+
+A [=range-request optimized font=] should not use any kind of compression other than [[!RFC7932]] (commonly referred to as "Brotli") compression.
+
+If Brotli compression is used in a [=range-request optimized font=], it should use only one <a href="https://datatracker.ietf.org/doc/html/rfc7932#section-9.2">meta-block</a>.
+
+If Brotli compression is used in a [=range-request optimized font=], its one meta-block should have the <a href="https://datatracker.ietf.org/doc/html/rfc7932#section-9.2"><code>ISUNCOMPRESSED</code></a> bit set to 1.
+
+Issue: We can probably do way better than just disabling all compression.
+
+<h3 id="font-organization-table-ordering">
+Table Ordering</h2>
+
+No two tables in a [=range-request optimized font=] should share a tag name.
+
+A [=range-request optimized font=] should have only one of these tables:
+- The <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/cff"><code>CFF</code></a> table
+- The <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/cff2"><code>CFF2</code></a> table
+- The <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/glyf"><code>glyf</code></a> table
+
+The term <dfn>outline table</dfn> is used to describe the one table which is either a <code>CFF</code> table, <code>CFF2</code> table, or <code>glyf</code> table.
+
+The [=outline table=] data in a [=range-request optimized font=] should lie at the end of the file.
+
+If a <code>CFF</code> table exists, the <a href="https://wwwimages2.adobe.com/content/dam/acom/en/devnet/font/pdfs/5176.CFF.pdf">CharString</a> data should lie at the end of the <code>CFF</code> table.
+
+If a <code>CFF2</code> table exists, the <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/cff2#charStrings">CharString</a> data should lie at the end of the <code>CFF2</code> table.
+
+Issue: https://github.com/w3c/IFT/issues/28 What about font collections?
+
+<h3 id="font-organization-glyph-independence">
+Glyph Independence</h2>
+
+Note: The goal of this section is to make every glyph independent from each other.
+
+A [=range-request optimized font=] should not use <a href="https://developer.apple.com/fonts/TrueType-Reference-Manual/RM03/Chap3.html">TrueType instructions</a>.
+
+Note: Some rasterizer software does not even execute these instructions in the first place.
+
+A [=range-request optimized font=] should not use <a href="https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6glyf.html">Compound glyphs</a>.
+
+Note: Compound glyphs can be flattened by inlining their component glyphs to become additional contours.
+
+A [=range-request optimized font=] should not use <a href="https://wwwimages2.adobe.com/content/dam/acom/en/devnet/font/pdfs/5176.CFF.pdf">Subroutines</a>.
+
+Note: CFF glyph CharStrings can be flattened by inlining subroutines to become additional CharString bytes.
+
```

Comment:
What if ... instead of including a list of things that _could be done_ to make a perfectly valid good (as in "efficiently designed") font file less efficient but usable with a range-request method, we could consider an alternative approach where we simply list range-request limitations and suggest that for TTF fonts that employ composite glyphs or for CFF fonts using subroutines other methods should be used - either a patch-subset or a full font download (that in many cases may be the most efficient option).

Reply:
That choice is up to the user / font service / website author. This is the specification for the range request method; if someone reads this and concludes that they don't want to use the range request method, that's their choice.

Reply:
Agree. However, making the spec more informative to aid to the understanding of all nuances that might affect this decision is a benefit, IMO. Since we already have a new issue #58 opened to address this topic, we should probably continue this discussion there.